src/codegen/property_body.rs

```diff
@@ -138,7 +138,10 @@ impl Builder {
         } else {
             ".unwrap()"
         };
-        body.push(Chunk::Custom(format!("value.get(){}", unwrap)));
+        body.push(Chunk::Custom(format!(
+            "value.get().expect(\"Return Value for property `{}` getter\"){}",
```

Comment:
This error not tell about actual reason.
IMHO just `unwrap` give better error.

Reply:
Yes its does... somewhat. See this discussion: https://github.com/gtk-rs/gir/pull/825#pullrequestreview-272474063 (you will need to click "Show resolved").

Reply:
Sorry, I missed that discussion.